<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QWidget" name="RegisterWindow">
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="loginInput">
     <property name="placeholderText">
      <string>Логин</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordInput">
     <property name="placeholderText">
      <string>Пароль</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameInput">
     <property name="placeholderText">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="surnameInput">
     <property name="placeholderText">
      <string>Фамилия</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="registerButton">
     <property name="text">
      <string>Зарегистрироваться</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

